.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "eslint-config-hapi"
```

Comment:
This will enforce `Stroustrup` style braces:
```if (){
}
else {

}```
which are an abomination, but which are also Hapi Standard (https://github.com/continuationlabs/eslint-config-hapi/blob/master/lib/index.js#L24).  I'm adding `'rules': {'brace-style': ['warn', '1tbs']}` to switch it to something more sane in my personal projects.  Thoughts on adding here too?